components/eam/src/chemistry/modal_aero/aero_model.F90

```diff
@@ -1131,42 +787,13 @@ subroutine aero_model_drydep  ( state, pbuf, obklen, ustar, cam_in, dt, cam_out,
              call outfld( trim(cnst_name(mm))//'DTQ', ptend%q(:,:,mm), pcols, lchnk)
              aerdepdryis(:ncol,mm) = sflx(:ncol)
 
-          else if ((lphase == 1) .and. (lspec == nspec_amode(m)+1)) then  ! aerosol water
-             ! use pvprogseasalts instead (means making the top level 0)
-             pvmzaer(:ncol,1)=0._r8
-             pvmzaer(:ncol,2:pverp) = vlc_dry(:ncol,:,jvlc)
-
-             if(.true.) then ! use phil's method
-             !      convert from meters/sec to pascals/sec
-             !      pvprogseasalts(:,1) is assumed zero, use density from layer above in conversion
-                pvmzaer(:ncol,2:pverp) = pvmzaer(:ncol,2:pverp) * rho(:ncol,:)*gravit
-
-             !      calculate the tendencies and sfc fluxes from the above velocities
-                call dust_sediment_tend( &
-                     ncol,             dt,       state%pint(:,:), state%pmid, state%pdel, state%t , &
-                     qaerwat(:,:,mm),  pvmzaer,  dqdt_tmp(:,:), sflx  )
-             else   !use charlie's method
-                call d3ddflux( ncol, vlc_dry(:,:,jvlc), qaerwat(:,:,mm), state%pmid, &
-                               state%pdel, tvs, sflx, dqdt_tmp(:,:), dt )
-             endif
-
-             ! apportion dry deposition into turb and gravitational settling for tapes
-             do i=1,ncol
-                if (vlc_dry(i,pver,jvlc) .ne. 0._r8) then
-                   dep_trb(i)=sflx(i)*vlc_trb(i,jvlc)/vlc_dry(i,pver,jvlc)
-                   dep_grv(i)=sflx(i)*vlc_grv(i,pver,jvlc)/vlc_dry(i,pver,jvlc)
-                endif
-             enddo
-
-             qaerwat(1:ncol,:,mm) = qaerwat(1:ncol,:,mm) + dqdt_tmp(1:ncol,:) * dt
-
           else  ! lphase == 2
              ! use pvprogseasalts instead (means making the top level 0)
              pvmzaer(:ncol,1)=0._r8
              pvmzaer(:ncol,2:pverp) = vlc_dry(:ncol,:,jvlc)
              fldcw => qqcw_get_field(pbuf, mm,lchnk)
 
-             if(.true.) then ! use phil's method
+!             if(.true.) then ! use phil's method
```

Comment:
if you leave this in as a comment, why not leave the "charlie's method" code below as comments, too?

Reply:
I leave this comment just to locate the code that can be separated into a subroutine as it is used a few times. Will clean it when I refactor this subroutine.